app/assets/javascripts/react/components/CirculationKiosk.js

```diff
@@ -0,0 +1,25 @@
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+import Kiosk from './presentational/Circulation/Kiosk';
+import * as actions from '../actions';
+
+// Top level mapping of the application state to properties of component that is being
+// connected.
+export const mapStateToProps = (state) => {
+  return {
+    slides: state.kiosk.slides,
+    url: state.kiosk.url,
+    is_fetching_slides: state.touch.is_fetching_slides,
+    show_nav: false,
```

Comment:
You want to set the default values for state the related reducer, and then pull the current value here as state.circ.show_nav.